packages/frontend/src/components/PageLayout/Selected.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect, useContext, createContext } from 'react';
+
+type ContextType = any;
+const SelectedContext = createContext<ContextType>({});
+
+export const useSelected = () => {
```

Comment:
`useSelected` er veldig vagt, det er for øvrig også navnet på filen.

Reply:
Har du forslag til bedre navn? `BulkSelection` og `useBulkSelection` kanskje?

Reply:
Oppdatert med bedre navngiving ref diksjon på Slack.